templates/activity/parts/activity-effects.hbs

```diff
@@ -1,29 +1,38 @@
 <fieldset>
-    <legend>{{ localize "DND5E.ACTIVITY.FIELDS.effects.label" }}</legend>
-    <multi-select name="appliedEffects">
-        {{ selectOptions allEffects }}
-    </multi-select>
-    <ul class="separated-list effects-list">
+    <legend>
+        <span>{{ localize "DND5E.ACTIVITY.FIELDS.effects.label" }}</span>
+        <button type="button" class="unbutton control-button" data-action="addEffect"
+                data-tooltip="DND5E.EFFECT.Action.Create"
+                aria-label="{{ localize "DND5E.EFFECT.Action.Create" }}">
+            <i class="fas fa-plus" inert></i>
+        </button>
+    </legend>
+    <div class="form-group">
+        <label>{{ localize "DND5E.EFFECT.Label" }}</label>
+        <div class="form-fields">
+            <multi-select name="appliedEffects" class="hidden-tags">
+                {{ selectOptions allEffects }}
+            </multi-select>
+        </div>
+    </div>
+    <ul class="separated-list effects-list unlist">
         {{#each appliedEffects}}
         <li data-index="{{ @index }}" data-effect-id="{{ effect.id }}" data-expand-id="effects.{{ effect.id }}">
             {{> ".effect" }}
         </li>
         {{/each}}
     </ul>
-    <button type="button" class="unbutton" data-action="addEffect">
-        {{ localize "DND5E.EFFECT.Action.Create" }}
-    </button>
 </fieldset>
 
 {{#*inline ".effect"}}
 <div class="details flexrow">
     <img class="gold-icon" src="{{ effect.img }}" alt="{{ effect.name }}">
     {{{ dnd5e-linkForUuid effect.uuid }}}
     <div class="list-controls flexrow">
-        <button type="button" class="unbutton" data-action="deleteEffect"
+        <button type="button" class="unbutton control-button" data-action="deleteEffect"
```

Comment:
The one issue with removing the `<multi-select>` is there is no longer a method to unassociated an effect with an activity without deleting it. That could be annoying if you accidentally associate one, the only way to fix it would be to delete either the activity or the effect.

Perhaps we could replace the standard `deleteDialog` in that case with one that gives the option of just removing from the list. Or we could have the default behavior just remove it from the list and when Shift is held down swap over to triggering effect deletion (with an icon swap like we have in the chat log).

Reply:
Added a separate 'dissociate' control since we have a control list, and it's a slightly less involved solution.